.template/addons/docker/Dockerfile.tt

```diff
@@ -31,7 +31,7 @@ ENV LANG="en_US.UTF-8" \
     LANGUAGE="en_US:en"
 
 RUN apt-get update -qq && \
-    apt-get install -y --no-install-recommends apt-transport-https curl gnupg
+    apt-get install -y --no-install-recommends apt-transport-https curl gnupg net-tools
```

Comment:
What is `net-tools` for?

Reply:
We need to install this package to use netstat. The serverspec use this one to check if the port is listening.